docs/configuration/configuration.md

```diff
@@ -564,6 +564,7 @@ address with relabeling.
 The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_gce_instance_name`: the name of the instance
+* `__meta_gce_machine_type`: the machine type URL of the instance
```

Comment:
Are you sure this is a URL?

Reply:
The godoc says it is full or partial URL.

https://godoc.org/google.golang.org/api/compute/v1#Instance

>    // MachineType: Full or partial URL of the machine type resource to use
>    // for this instance, in the format:
>    // zones/zone/machineTypes/machine-type. This is provided by the client
>    // when the instance is created. For example, the following is a valid
>    // partial url to a predefined machine
>    // type:
>    // zones/us-central1-f/machineTypes/n1-standard-1

Here is my example.

```yaml
scrape_configs:
  - job_name: 'gce-node-exporter'
    gce_sd_configs:
    - project: alba-197510
      zone: us-west1-b
      refresh_interval: 5m
    relabel_configs:
    - source_labels: [__meta_gce_instance_name]
      regex: '(.*)'
      replacement: '${1}'
      target_label: instance
    - source_labels: [__meta_gce_machine_type]
      regex: '.*?/([^/]*)$'
      replacement: '${1}'
      target_label: machine_type
    - source_labels: [__meta_gce_private_ip]
      regex: '(.*)'
      replacement: '${1}:9100'
      target_label: __address__
```

![label](https://user-images.githubusercontent.com/3808731/38160611-99d662e0-34fb-11e8-9c83-9a98937e1bb6.png)

Reply:
A partial URL is not a URL, it's an arbitrary string.